src/Stryker.Core/Stryker.Core/Reporters/Progress/ProgressBarReporter.cs

```diff
@@ -74,7 +79,36 @@ public void ReportRunTest()
 
             _testsProgressLogger.ReplaceLog(LoggingFormat,
                                             stringBuilder, _numberOfTestsRun, _totalNumberOfTests,
-                                            totalNumberOfTestsPercentage);
+                                            totalNumberOfTestsPercentage, RemainingTime());
+        }
+
+        private string RemainingTime()
+        {
+            if (_totalNumberOfTests == 0 || _numberOfTestsRun == 0)
+            {
+                return "NA";
+            }
+
+            var elapsed = (DateTime.UtcNow - _startTime).TotalSeconds;
+            var remaining = (_totalNumberOfTests - _numberOfTestsRun) * elapsed / _numberOfTestsRun;
```

Comment:
The calculation is pretty accurate but could be better. We could add a little extra istimated time for future timeouts? At the start of my testrun it estimated too low (13 minutes instead of 18 actual), and started to get better later when some timeouts occured. Maybe you could experiment with this?

Reply:
Current calculation is naive and basic. I can devise a more sophisticated model, but tuning will be tricky as timeout percentage may significantly vary. If you have only a few mutants, timeouts will significantly impact the estimated remaining time.
Alternatively, if you have a lot of mutants, estimate should be pretty good when you reach 10% of testing. And as this feature is mostly useful for larger projects, I propose to keep this algorithm as is and open an issue for a more elaborated one, where the model can be discussed

Reply:
True, it doesn't have to be perfect so this is a very good start. I was thinking just now: by using a timestamp as the starttime the estimation would be way off if the computer would go to sleep for several hours and continued later. Is there a way we could exclude the sleep time?

Reply:
Good point. I could use a strategy similar to the one used to monitor timeouts, but this would probably badly impair precision. But I have an alternate strategy I need to try first.

Reply:
I switched to Stopwatch (instead DateTime). Less accurate, but will compensate for sleep phase. But it depends on the quality of implementation and IIRC it may be incorrect for some AMD CPUs when they enter low power modes